test/network/networkWrapper.test.js

```diff
@@ -10,8 +10,10 @@ var
 
 describe('Network Wrapper', function () {
   afterEach(function () {
-    window = undefined; // eslint-disable-line
-    WebSocket = undefined; // eslint-disable-line
+    window = undefined;
+    /*eslint no-native-reassign: "off"*/
```

Comment:
Nitpicking: this disables the rule for the entire file. Prefer the following:
`// eslint-disable-next-line no-native-reassign, no-global-assign`